wasm.js

```diff
@@ -0,0 +1,174 @@
+/*
+ * This file is part of the 0chain @zerochain/0chain distribution (https://github.com/0chain/client-sdk).
+ * Copyright (c) 2018 0chain LLC.
+ *
+ * 0chain @zerochain/0chain program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+'use strict';
+
+const utils = require('./utils');
+const g = global || window || self;
+
+async function blsSign(hash) {
+    if (!bridge.bls?.secretKey) {
+        const err = 'err: bls.secretKey is not initialized';
+        console.error(err);
+        throw err;
```

Comment:
If we are going to throw, we better throw a proper Error instance. It's much friendlier to clients:

```suggestion
        const errMsg = 'err: bls.secretKey is not initialized';
        console.error(errMsg);
        throw new Error(errMsg);
```

Reply:
Same applies everywhere else.